nuxeo-core/nuxeo-core-storage-mongodb/src/main/java/org/nuxeo/ecm/core/storage/mongodb/MongoDBChecker.java

```diff
@@ -57,28 +65,34 @@ public boolean accepts(ConfigurationGenerator cg) {
 
     @Override
     public void check(ConfigurationGenerator cg) throws ConfigurationException {
-        MongoClient ret = null;
-        String serverName = cg.getUserConfig().getProperty(ConfigurationGenerator.PARAM_MONGODB_SERVER);
-        String dbName = cg.getUserConfig().getProperty(ConfigurationGenerator.PARAM_MONGODB_NAME);
-
-        MongoClientOptions.Builder optionsBuilder = MongoClientOptions.builder()
-                                                                      .serverSelectionTimeout(
-                                                                              (int) TimeUnit.SECONDS.toMillis(getCheckTimeoutInSeconds(cg)))
-                                                                      .description("Nuxeo DB Check");
-        if (serverName.startsWith("mongodb://")) {
-            // allow mongodb:// URI syntax for the server, to pass everything in one string
-            ret = new MongoClient(new MongoClientURI(serverName, optionsBuilder));
-        } else {
-            ret = new MongoClient(new ServerAddress(serverName), optionsBuilder.build());
+        File configFile = new File(cg.getConfigDir(), CONFIG_NAME);
+        if (configFile.exists()) {
```

Comment:
Please inverse the logic to do early-exit if the config file doesn't exist (and log something at debug leve).